src/app.py

```diff
@@ -14,15 +16,88 @@
 app = FastAPI(debug=DEBUG,
               title='Linktious')
 
+
+# Dependency
+def get_db():
+    db = base.SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+# TODO: add validation to routes
+# TODO: return status codes
 @app.get("/")
 def read_root():
     return {"status": "OK"}
 
+
 @app.get("/health")
 def health_check():
     return {"status": "Ok"}
 
 
+@app.get("/teams", response_model=List[schema.Team])
+def get_team(db: Session = Depends(get_db)):
+    return crud.get_teams(db=db)
+
+
+@app.get("/users/{user_email}", response_model=schema.User)
+def get_user_by_email(user_email: str, db: Session = Depends(get_db)):
+    return crud.get_user_by_email(db=db, user_email=user_email)
+
+
+@app.get("/users/{user_id}", response_model=schema.UserBasicInfo)
+def get_user_basic_info_by_id(user_id: int, db: Session = Depends(get_db)):
+    return crud.get_user_by_id(db=db, user_id=user_id)
+
+
+@app.put("/users/{user_id}/set_main_board", response_model=schema.User)
+def set_user_main_board(user_id: int, board_id: int, db: Session = Depends(get_db)):
+    return crud.set_user_main_board(db=db, user_id=user_id, board_id=board_id)
+
+
+@app.put("/users/{user_id}/set_favorite_boards", response_model=schema.User)
+def set_user_favorite_boards(user_id: int, boards_ids: List[int], db: Session = Depends(get_db)):
+    return crud.set_user_favorite_boards(db=db, user_id=user_id, boards_ids=boards_ids)
+
+
+@app.get("/labels", response_model=List[schema.Label])
+def get_labels(db: Session = Depends(get_db)):
+    return crud.get_labels(db=db)
+
+
+@app.post("/labels", response_model=schema.Label)
+def create_label(label: schema.LabelCreate, db: Session = Depends(get_db)):
+    return crud.create_label(db=db, label=label)
+
+
+@app.get("/links", response_model=List[schema.Link])
+def get_links(db: Session = Depends(get_db)):
+    return crud.get_links(db=db)
+
+
+@app.post("/links", response_model=schema.Link)
+def create_link(link: schema.LinkCreate, db: Session = Depends(get_db)):

```

Comment:
what do you think about adding black as the code formatter?
https://black.readthedocs.io/en/stable/the_black_code_style.html#code-style
and its automatic!
I really like the fact that each param is in it's own line:
```
def create_link(
   link: schema.LinkCreate,
   db: Session = Depends(get_db),
):
    pass
```

Reply:
Looks greate but I prefer to configure the project to use black in a different issue